src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -434,6 +445,9 @@ static inline void cavs_pm_runtime_dis_dsp_pg(uint32_t index)
 		if (tries == 0)
 			tr_err(&power_tr, "cavs_pm_runtime_dis_dsp_pg(): failed to power up core %d",
 			       index);
+#else
+		z_wrapper_cpu_enable_secondary_core(index);
+#endif
 		pprd->dsp_client_bitmap[index] |= flag;
 	}
 #endif
```

Comment:
Would be nice
```suggestion
#endif /* CAVS_VERSION >= CAVS_VERSION_1_8 */
```

Reply:
@marc-hb Not related to this patch, right.